### ISSUE TITLE
Implement randomization at index

### ISSUE THREAD

**Issue by arthurhjorth** [arthurhjorth/simple-chatbot#4]
When people first land on the chat app from prolific or qualtrics or whatever, we need to randomize them into the various conditions.

My understanding so far is a 2 x 2 with human-human | human-chatbot on one axis and correct/information | incorrect/misinformation on the other.

For the human-human conditions, I am not sure how misinformation will be implemented. I guess through instructions? So we need to implement a variety of instructions for each of the four conditions, and we need a template for displaying them.

For the human-chatbot conditions we just need two different system instructions; one that prompts the chatbot to persuade correctly and one for incorrectly.

**Comment by zahoorgtech** [comment]
Yes, using clear and specific instructions is the right approach. We can use some scripted prompts too that subtly introduce misinformation and to guide the conversation without forcing it unnaturally.
I think a potential challenge regarding misinformation is to ensure that participants are not misled beyond the scope of the study's experimental setting.
